### PR TITLE
add dataframe pyarrow update

### PR DESCRIPTION
**Pull request by glemaitre**
@jorisvandenbossche would you mind to draft a short abstract :)

### CODE REVIEW

**Comment by glemaitre** [comment]
Proposing a suggestion is fine.
I will accept it and then merge it.

File: content/pages/2022/maintainers.md [hunk 1]
```diff
@@ -77,6 +77,12 @@ existing and upcoming challenges.
 - Tools: What are the existing tools in the ecosystem helping us achieve the
   above goals, and what do we need to develop?
 
+#### DataFrame API & PyArrow Update
```
Comment (jorisvandenbossche) [inline content/pages/2022/maintainers.md:80]:
```suggestion
#### Interoperability in the DataFrame ladncape: DataFrame API & PyArrow Update
```

File: content/pages/2022/maintainers.md [hunk 2]
```diff
@@ -77,6 +77,12 @@ existing and upcoming challenges.
...
+**Chair: Joris Van den Bossche**
+**Time: <TBA>**
+
+A small and short abstract.
```
Comment (jorisvandenbossche) [inline content/pages/2022/maintainers.md:84]:
```suggestion
In this session, we want to share some updates on the DataFrame ecosystem, the DataFrame interchange protocol and Arrow interface, and the integration with different libraries. 
```

Comment (jorisvandenbossche) [reply content/pages/2022/maintainers.md:84]: (will try to make a longer version later today)

File: content/pages/2022/maintainers.md [hunk 3]
```diff
@@ -77,6 +77,13 @@ existing and upcoming challenges.
 - Tools: What are the existing tools in the ecosystem helping us achieve the
   above goals, and what do we need to develop?
 
+#### Interoperability in the DataFrame landscape: DataFrame API & PyArrow Update
+**Chair: Joris Van den Bossche**
+**Time: <TBA>**
+
+In this session, we want to share some updates on the DataFrame ecosystem, the DataFrame
+interchange protocol and Arrow interface, and the integration with different libraries.
```
Comment (jorisvandenbossche) [inline content/pages/2022/maintainers.md:85]:
```suggestion
In this session, we want to share some updates on the DataFrame ecosystem: the DataFrame
interchange protocol (https://data-apis.org/dataframe-protocol/latest/purpose_and_scope.html)
and Arrow C Data interface (https://arrow.apache.org/docs/format/CDataInterface.html),
and the integration of those interoperability protocols with different libraries. Further, we
want to have an open conversation about challenges and requirements related to DataFrame
interoperability and supporting multiple DataFrame libraries in projects.
```

**Comment by glemaitre** [comment]
Thanks @jorisvandenbossche I am going to merge now.

**Pull request merged**